app/models/user.py

```diff
@@ -35,3 +54,23 @@ def verify_auth_token(token):
             return None
         user = User.query.get(data['name'])
         return user
+
+    def __repr__(self):
+        return '{}'.format(self.id)
+
+
+class Task(Base):
+
+    __tablename__ = 'tasks'
+
+    title = Column(String(200), unique=True, nullable=False)
+    status = Column(String(15), nullable=False)
+
+    users = relationship("User", secondary="association")
+
+    def assign_user(self, userid):
+        self.association.append(Association(user_id = userid,
+            task_id = self.id))
+
+    def __repr__(self):
+        return '{}'.format(self.title)
```

Comment:
in one case repr works with id, in other with name. please use one approach for all models.